uint8 head_leftright
uint8 head_updown
uint8 head_lefteye
uint8 head_righteye
uint8 head_finish
